recipe/migrations/pypy38.yaml

```diff
@@ -0,0 +1,52 @@
+migrator_ts: 1647123563
+__migrator:
+    migration_number: 1
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.6.* *_73_pypy
+            - 3.7.* *_73_pypy
+            - 3.8.* *_73_pypy
+            - 3.9.* *_73_pypy
+    paused: False
+    longterm: True
+    check_solvable: True
+    exclude_pinned_pkgs: False
+    pr_limit: 1
+    bump_number: 1
+    wait_for:
+      - python310
```

Comment:
Does this mean we wait for the whole migration to complete or for the feedstock to have already undergone the Python 3.10 migration?

Reply:
My understanding is that it's per-feedstock, see https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/migrators/migration_yaml.py#L176-L198 and https://github.com/regro/cf-scripts/pull/1405/files